common/iptables.go

```diff
@@ -0,0 +1,7 @@
+package common
```

Comment:
Generally `common` is something we'd rather have less of - could this go in `net` instead?  Or maybe just local to `npc` since it doesn't appear to be used outside.